weld/conf.rs

```diff
@@ -86,17 +95,31 @@ fn parse_passes(s: &str) -> WeldResult<Vec<Pass>> {
     Ok(result)
 }
 
+/// Parse an LLVM optimization level.
+fn parse_llvm_optimization_level(s: &str) -> WeldResult<u32> {
+    match s.parse::<u32>() {
+        Ok(v) if v <= 2 => Ok(v),
```

Comment:
Maybe make this 4 or something? Since LLVM also supports O2 and O3.

We may (eventually) want to make our own passes too, that e.g., disable loop vectorization and stuff since that will never happen.

Reply:
It actually only takes up to 3, see http://llvm.org/doxygen/classllvm_1_1PassManagerBuilder.html#a52b96ef67bb93fa3a7b461fe2b73bb4f. We should add other configs if we want other sets of passes.